docs/reference/supervisor.md

```diff
@@ -820,6 +825,7 @@ public class Node {
 ```MATLAB
 position = wb_supervisor_node_get_position(node)
 orientation = wb_supervisor_node_get_orientation(node)
+pose = wb_supervisor_node_get_pose(node, from_node=0)
```

Comment:
I am not sure the default value of 0 is supported in MATLAB. Did you check it?

Reply:
I think it is better now

Reply:
Yes, it looks very good.